composer/loggers/mlflow_logger.py

```diff
@@ -346,6 +346,55 @@ def log_model(self, flavor: Literal['transformers'], **kwargs):
             else:
                 raise NotImplementedError(f'flavor {flavor} not supported.')
 
+    def create_model_version(
+        self,
+        model_uri: str,
+        name: str,
+        await_creation_for: int = 300,
+        tags: Optional[Dict[str, Any]] = None,
+    ):
+        """Similar to MLflow's create_model_version, but will also create the initial model if it does not exist.
```

Comment:
Why do we need to create the initial model if it does not exist?

Reply:
Ah, I looked at the mlflow code more and it make sense to me. It looks like it's more similar to `register_model`. It seems like the only difference is that it uses the run_id directly from the logger instead of parsing it from model_uri?

Maybe rename the function to `register_model_with_run_id`?

Reply:
yeah, thats fair. I called it this cause originally it was just a wrapper around `create_model_version`, but then I discovered I had to do the try/except thing to create if not exists. Will update.

Reply:
Can you update the docstring to just say that it's similar to register model?